rigex-front-end/src/components/rigScheduler/Header.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Faker from 'faker';
+
+import './headerStyle.scss';
+
+import Mockup from '../../assets/Rig-Scheduler-Mockup.png';
+const CompanyLogo = Faker.image.business();
+
+const SchedulerHeader = () => {
+  return (
+    <div className="grid-container">
+      <div className="company-logo">
+        <div class="logo-container">
+          <img
+            className="company-logo-image"
+            alt="company logo"
+            src={CompanyLogo}
+          ></img>
+        </div>
+      </div>
+      <div className="full-title">
+        <span className="title-text">Rig Scheduler</span>
+        <span className="sub-title-text">
+          Achieve on schedule and on budget rig planning
+        </span>
+      </div>
+      <div className="computer-image">
+        <img className="mockup" alt="Rig Scheduler Mockup" src={Mockup} />
```

Comment:
this src should also be using faker (can be same company logo I don't really care) and then remove the import for this along with the png

Reply:
fixed